Automattic/wp-calypso
# Add/high quality background image

## Proposed Changes

Add higher quality for background images for LiB and NL. The images are heavier but only loaded on larger screens.

## Testing Instructions

- Pull this branch and run yarn start
- Navigate to `/setup/intro?flow=link-in-bio` or `/setup/intro?flow=newsletter`
- Check if on mobile the background image for intro and processing steps has `-desktop` at the end of the file name for desktop and don't have it for mobile.
- Visually they should also look better, but could be hard to check the difference.


Related to #67451
Fixes #67451
